### PR TITLE
Adjusted example configs to show example of JWT_SECRET and CORS_ORIGINS

### PR DESCRIPTION
**Pull request by jivanyatra**
Hello!

I saw you already added the tips on generating the JWT_SECRET and what to put for CORS_ORIGIN. I just updated the example configs to show what they might look like. Just a suggestion.

I'm also happy to add another example config. My use case is using docker compose to deploy the frontend and backend containers, but using my local nginx installation with a subdomain to proxy to those containers. It's not very complicated, so not sure how much use it actually will be for self-hosters. Please let me know what you think!

### CODE REVIEW

**Comment by jivanyatra** [comment]
@enchant97 just pinging in case you want to merge or close.

File: site/content/docs/setup/examples/postgres-docker-compose.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ services:
       - backend_data:/data
     environment:
       # !!! REPLACE These !!!
-      JWT_SECRET: "bXktc2VjcmV0"
-      CORS_ORIGINS: "*"
+      JWT_SECRET: "qDSFr/3Mq9zX+CVYtlSjpmJyYMNXYGBNLDSu51RAuNQ="
```
Comment (enchant97) [inline site/content/docs/setup/examples/postgres-docker-compose.yml:17]: can we keep the jwt secret as it was?

Comment (jivanyatra) [reply site/content/docs/setup/examples/postgres-docker-compose.yml:17]: I'll do anyway you like!

I do think it often helps to show users what it *should* look like, but then again that means lazy folks will use what you give them. I can revert that change and update the PR

Comment (enchant97) [reply site/content/docs/setup/examples/postgres-docker-compose.yml:17]: I think it would be best to be the original. It is still a valid secret.

Comment (jivanyatra) [reply site/content/docs/setup/examples/postgres-docker-compose.yml]: Reverted the JWT change!

Comment (enchant97) [reply site/content/docs/setup/examples/postgres-docker-compose.yml]: Thanks, will merge now!

**Pull request merged**